test/components/generators/chat/test_azure.py

```diff
@@ -331,6 +333,31 @@ def test_live_run_with_tools(self, tools):
         assert tool_call.arguments == {"city": "Paris"}
         assert message.meta["finish_reason"] == "tool_calls"
 
+    @pytest.mark.skipif(
+        not os.environ.get("AZURE_OPENAI_API_KEY", None),
+        reason="Export an env var called AZURE_OPENAI_API_KEY containing the Azure OpenAI API key to run this test.",
+    )
+    @pytest.mark.integration
+    def test_live_run_with_response_format(self):
+        class CalendarEvent(BaseModel):
+            name: str
+            date: str
+            location: str
+
+        chat_messages = [ChatMessage.from_user("Describe the 20th Nobel Peace Prize.")]
```

Comment:
Let's use a text from which the model can actually extract all the required attributes.